sagenb/data/sage/html/history.html

```diff
@@ -3,13 +3,13 @@
 {% block title %}{{ gettext('Sage: History for %(u)s', u=username) }}{% endblock %}
 
 {% block javascript %}
-<script type="text/javascript" src="/javascript/sage/js/ws_list.js"></script>
+<script type="text/javascript" src="{{request.url_root}}/javascript/sage/js/ws_list.js"></script>
 {% endblock %}
 
 {% block page_id %}history-page{% endblock %}
 
 {% block body %}
-    <pre>{{ text | escape }}</pre>
```

Comment:
There were a bunch of these and other alternatives that were causing page rendering errors.  I no longer remember the detailed problems, but in the first commit where I started doing this there should be a comment.  I will have to dig.